mesos-docker/run/run.sh

```diff
@@ -198,6 +206,31 @@ function start_slaves {
       HADOOP_VOLUME=
     fi
 
+
+    if [[ -n $SLAVES_CONFIG_FILE ]]; then
+      if [[ -f $SLAVES_CONFIG_FILE ]]; then
+        . cfg.sh
```

Comment:
Same as the previous comment. This command checks that `jq` is available, but it is fairly late in the execution path. The check that `jq` is available should be done before doing any real work.